packages/sfui/frameworks/nuxt/src/module.ts

```diff
@@ -0,0 +1,69 @@
+import { defineNuxtModule, addComponent, addImportsSources, installModule, addImports } from '@nuxt/kit';
+import { NuxtOptions } from '@nuxt/schema';
+import * as storefrontUi from '@storefront-ui/vue';
+import { tailwindConfig } from '@storefront-ui/vue/tailwind-config';
+
+// Module options TypeScript interface definition
+export interface ModuleOptions {
+  contentPath?: string;
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: 'storefront-ui',
+    configKey: 'storefrontUi',
+    compatibility: {
+      nuxt: '>=3.0.0',
+    },
+  },
+  // Default configuration options of the Nuxt module
+  defaults: {
+    contentPath: './node_modules/@storefront-ui/vue/**/*.mjs',
+  },
+  async setup(options, nuxt) {
+    const { contentPath } = options;
+    const nuxtOptions = nuxt.options;
+
+    nuxtOptions.tailwindcss = {
+      ...nuxtOptions.tailwindcss,
+      config: {
+        presets: [tailwindConfig],
+        content: [contentPath ?? ''],
+        ...nuxtOptions.tailwindcss?.config,
+        // if content is already defined, we need to merge it with the new one
+        ...(Array.isArray(nuxtOptions.tailwindcss?.config?.content) && contentPath
+          ? { content: [...nuxtOptions.tailwindcss?.config?.content!, contentPath] }
+          : {}),
```

Comment:
I'd propose to rewrite this part in regular ifs - imo it's a bit more readable 

```suggestion
   const content = [];
   
   if (Array.isArray(nuxtOptions.tailwindcss?.config?.content?.files)) {
     content.push(...nuxtOptions.tailwindcss?.config?.content?.files);
   }
   if (Array.isArray(nuxtOptions.tailwindcss?.config?.content)) {
     content.push(...nuxtOptions.tailwindcss?.config?.content);
   }
   if (contentPath) content.push(contentPath);

    nuxtOptions.tailwindcss = {
      ...nuxtOptions.tailwindcss,
      config: {
        presets: [tailwindConfig],
        content,
```

Reply:
100a3271b45a8e6ba85992e423284fd05a8a7056